tests/benchdnn/inputs/graph/complex_fusion/mha/sdpa-compressed-k-int8-gs32.json

```diff
@@ -32,13 +32,13 @@
             "value": [
               1,
               1,
-              32,
-              1
+              1,
+              32
```

Comment:
What's the purpose of changing the json file? My understanding is that all of these shape/attribute changes can be done through benchdnn command line.

Reply:
Thanks for reminding. I've moved the test case to the batch file with cml rewriting.